From 05ee941ff5503782fbbb0d4c5d085462d998d169 Mon Sep 17 00:00:00 2001
From: Mingcong Bai <jeffbai@aosc.io>
Date: Sun, 6 Sep 2026 15:14:58 +0800
Subject: [PATCH 7/8] AOSCOS: fix(kicker): do not show panel preview as
 background for launcher icon

The current implementation does not take currentColor hint into
consideration, making the user-specified icon practically invisible when
using a dark theme.

Signed-off-by: Mingcong Bai <jeffbai@aosc.io>
---
 applets/kicker/ConfigGeneral.qml | 2 --
 1 file changed, 2 deletions(-)

diff --git a/applets/kicker/ConfigGeneral.qml b/applets/kicker/ConfigGeneral.qml
index 28e983bddf..f518cb9bb3 100644
--- a/applets/kicker/ConfigGeneral.qml
+++ b/applets/kicker/ConfigGeneral.qml
@@ -110,8 +110,6 @@ KCMUtils.SimpleKCM {
             KSvg.FrameSvgItem {
                 id: previewFrame
                 anchors.centerIn: parent
-                imagePath: Plasmoid.location === PlasmaCore.Types.Vertical || Plasmoid.location === PlasmaCore.Types.Horizontal
-                        ? "widgets/panel-background" : "widgets/background"
                 width: Kirigami.Units.iconSizes.large + fixedMargins.left + fixedMargins.right
                 height: Kirigami.Units.iconSizes.large + fixedMargins.top + fixedMargins.bottom
 
-- 
2.55.0

